python/pyarrow/scalar.pxi

```diff
@@ -731,6 +731,57 @@ cdef class LargeListValue(ArrayValue):
         return result
 
 
+cdef class MapValue(ArrayValue):
```

Comment:
Perhaps add a test in `test_scalars.py`.

Reply:
done